built-in-references/kubernetes/container-allowed-images/constraint.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: constraints.gatekeeper.sh/v1beta1
+kind: K8sAzureContainerAllowedImages
+metadata:
```

Comment:
Does this content need a README describing how to enter the private preview like KubernetesService has?  If someone sees these files and wants to use them how do they get started?  The top level README just links to the language and a page about what AKS engine is

Reply:
Good point. Thank you.
We are going public preview for AKS Engine, will point to the documentation in both cases. I will also mentioned AKS is in private preview and AKS Engine is in public preview.

Reply:
Updated README.